src/scripts/glue_jobs/process_access_record.py

```diff
@@ -51,31 +51,30 @@ def get_dynamic_frame(connection_type, file_format, source_path, glue_context):
     return dynamic_frame
 
 
-# There are many fields in access records, however we need some of them for further processing
 def apply_mapping(dynamic_frame):
     mapped_dynamic_frame = ApplyMapping.apply(
         frame=dynamic_frame,
         mappings=[
             ("payload.sessionId", "string", "SESSION_ID", "string"),
-            ("payload.timestamp", "long", "TIMESTAMP", "long"),
-            ("payload.userId", "int", "USER_ID", "int"),
+            ("payload.timestamp", "bigint", "TIMESTAMP", "bigint"),
```

Comment:
can we use a table type of TIME_STAMP and then drop both DATE and DATE_AND_TIME.